docs/testing-tool.rst

```diff
@@ -7,85 +7,125 @@ Use the KSQL testing tool to test a set of KSQL statements. The KSQL testing too
 is a command line utility that enables testing KSQL statements without requiring any infrastructure, like |ak-tm| and KSQL clusters.
 The KSQL testing tool is a great way to design your KSQL pipeline and ensure the expected results are generated.
 You can collaborate on designing your KSQL statements by sharing the test files.
-You provide a JSON file that describes a set of KSQL statements, along with the input data and expected output data.
-Run the testing tool from a terminal and pass the test file as a parameter.
+To test a set of KSQL statements, you provide three files, one file containing the KSQL statements and two JSON files containing the input records and the expected output records.
 
 .. code:: bash
 
-    ksql-testing-tool /path/to/the/test/file.json
+    NAME
+            ksql-test-runner - The KSQL testing tool
+
+    SYNOPSIS
+            ksql-test-runner {--input-file | -i} <inputFile>
+                    {--output-file | -o} <outputFile>
+                    {--sql-file | -s} <statementsFile>
+
+    OPTIONS
+            --input-file <inputFile>, -i <inputFile>
+                A JSON file containing the input records.
+
+                This option may occur a maximum of 1 times
```

Comment:
```suggestion
                This option may occur a maximum of 1 time
```

Reply:
Interesting, that's the message that is generated automatically by the airline package. I updated it in the docs anyway.